deployment/source-only-mode.md

```diff
@@ -0,0 +1,76 @@
+# Source Only Mode
+
+Since v1.18.0, Fluentd can launch with source-only mode.
+(Not supported on Windows)
+
+* In this mode, only input plugins runs.
+* During this mode, the events are stored in a temporary file buffer.
+* Sending `SIGWINCH` to the supervisor cancels this mode.
+  * Then, all plugins start to run, and the temporary file buffer starts to load.
+
+## How to launch Fluentd with source-only mode
+
+You can launch Fluentd with source-only mode in the following ways.
+
+* [Command Line Option - --with-source-only](command-line-option.md)
+* [System Configuration - with_source_only](system-config.md#with_source_only)
+
+## Temporary file buffer
+
+During source-only mode, the events are stored in a temporary file buffer.
+After `SIGWINCH` is sent to the supervisor and this mode is canceled, this buffer begins to load.
+
+By default, the file buffer path is as follows.
+
+* `/tmp/fluent/source-only-buffer/{Unique ID for the Fluentd instance}`
+
+You can confirm it in the following log output at startup.
+
+```
+[info]: #0 with-source-only: the emitted data will be stored in the buffer files under
+/tmp/fluent/source-only-buffer/bbd9006d-bc41-418b-b346-f80888641dda. You can send SIGWINCH to
+the supervisor process to cancel with-source-only mode and process data.
+```
+
+This file buffer is the buffer of [out_buffer](../output/buffer.md).
+It works with the default settings of `out_buffer` except for the following points:
+
+* `path` is automatically determined by default.
+* `overflow_action` is `drop_oldest_chunk` by default.
+
+If need, you can configure some options for the buffer in System Configuration.
+Please see the following for details.
+
+* [System Configuration - \<source_only_buffer\> section](system-config.md#less-than-source_only_buffer-greater-than-section).
+
+### Recovering
+
+If Fluentd stops with the temporary buffer left, you need to recover the buffer to launch Fluentd with source-only mode again.
+
+Note that a different path will be used each time unless you configure the temporary buffer path explicitly.
+In this case, you can recover the buffer as follows.
+
+1. Configure the left buffer path explicitly.
+1. Start Fluentd with source-only mode again.
+1. Send `SIGWINCH` to the supervisor to load the buffer.
+
+If this recovering is necessary, i.e., Fluentd stops with the temporary buffer left, the following warning log will be output.
```

Comment:
```suggestion
If this recovery is necessary, i.e., Fluentd stops with the temporary buffer in place, the following warning log will be output.
```

Reply:
Thanks, I will fix `recovering` to `recovery` entirely.

What is the problem with the following expression?

* `with the temporary buffer left`

Reply:
> What is the problem with the following expression?
> 
>     * `with the temporary buffer left`

Fixed! I use `remained` instead of `left`.